infra/README.md

```diff
@@ -56,7 +65,7 @@ gcloud builds submit \
 --tag $REGION-docker.pkg.dev/$PROJECT_ID/$AR_REPO_NAME/$IMAGE_NAME .
 ```
 
-3. Create secret for Cloud Run (Next.js app) container with [Secret Manager](Secret Manager).
+4. Create secret for Cloud Run (Next.js app) container with [Secret Manager](Secret Manager).
```

Comment:
It's not clear here what the secret is for, why are we telling them to create it?